lib/include/tree_sitter/api.h

```diff
@@ -26,13 +26,13 @@ extern "C" {
  * The Tree-sitter library is generally backwards-compatible with languages
  * generated using older CLI versions, but is not forwards-compatible.
  */
-#define TREE_SITTER_LANGUAGE_VERSION 14
+#define TREE_SITTER_LANGUAGE_VERSION 15
 
 /**
  * The earliest ABI version that is supported by the current version of the
  * library.
  */
-#define TREE_SITTER_MIN_COMPATIBLE_LANGUAGE_VERSION 13
+#define TREE_SITTER_MIN_COMPATIBLE_LANGUAGE_VERSION 14
```

Comment:
Is this necessary? A number of parsers in the wild are still on ABI 13.

Reply:
ABI 14 is more than two years old at this point

Reply:
So? What harm does this do?

Reply:
Ok, we can keep the runtime support for 13, but I'll remove allowing the CLI to generate ABI 13. If the compatibility code between 14 and 15 becomes too unwieldy with 13 in the mix later on, I'll revisit removing runtime support then.

Reply:
Yes, that is a good compromise.